File: ResourceManager/ViewModels/Books/EbookOptionsContentViewModel.cs
using ResourceManager.Configuration;
using ResourceManager.Helpers;

namespace ResourceManager.ViewModels
{
    public class EbookOptionsContentViewModel : PageContentViewModel
    {
        public EbookOptionsContentViewModel()
        {
            MenuTitle = "Options";
            MenuIcon = AssetManager.Instance.GetImage("Options.png");
            Title = "Ebook Options";
            PageColor = Constants.EbookPageColor;

            FolderPath = App.Settings.EbookSettings.FolderPath;
            SupportExtensions = App.Settings.EbookSettings.SupportExtensions;
            CoverPath = App.Settings.EbookSettings.CoverPath;
            CoverFileExtensions =  App.Settings.EbookSettings.CoverFileExtensions;
            DeleteSourceFile =  App.Settings.EbookSettings.DeleteSourceFile.ToString();
        }
        public string FolderPath { get; }
        public string SupportExtensions { get; }
        public string CoverPath { get; }
        public string CoverFileExtensions { get; }
        public string DeleteSourceFile { get; }
    }
}

File: ResourceManager/ViewModels/Books/EbookOptionsContentViewModel.vb
Imports ResourceManager.Configuration
Imports ResourceManager.Helpers

Namespace ResourceManager.ViewModels
    Public Class EbookOptionsContentViewModel
        Inherits PageContentViewModel
        Public Sub New()
            MenuTitle = "Options"
            MenuIcon = AssetManager.Instance.GetImage("Options.png")
            Title = "Ebook Options"
            PageColor = EbookPageColor

            FolderPath = Application.Settings.EbookSettings.FolderPath
            SupportExtensions = Application.Settings.EbookSettings.SupportExtensions
            CoverPath = Application.Settings.EbookSettings.CoverPath
            CoverFileExtensions = Application.Settings.EbookSettings.CoverFileExtensions
            DeleteSourceFile = Application.Settings.EbookSettings.DeleteSourceFile.ToString()
        End Sub
        Public ReadOnly Property FolderPath As String
        Public ReadOnly Property SupportExtensions As String
        Public ReadOnly Property CoverPath As String
        Public ReadOnly Property CoverFileExtensions As String
        Public ReadOnly Property DeleteSourceFile As String
    End Class
End Namespace
